backend/program/scrapers/orionoid.py

```diff
@@ -1,27 +1,23 @@
 """ Orionoid scraper module """
 from typing import Optional
-from pydantic import BaseModel
+
 from requests import ConnectTimeout
 from requests.exceptions import RequestException
+
 from utils.logger import logger
 from utils.request import RateLimitExceeded, RateLimiter, get
-from utils.settings import settings_manager
+from program.settings.manager import settings_manager
 from utils.parser import parser
 
 KEY_APP = "D3CH6HMX9KD9EMD68RXRCDUNBDJV5HRR"

```

Comment:
Is this really supposed to be here?

Reply:
the `KEY_APP` ? 

Yes. This is specific to Iceberg in regards with Orionoid. Users still need an api key for Orionoid to authenticate.